apps/sports-helsinki/src/domain/search/eventSearch/searchResultList/__tests__/ResultsInfo.test.tsx

```diff
@@ -4,18 +4,18 @@ import { render, screen } from '@/test-utils';
 import ResultsInfo from '../ResultsInfo';
 
 it('events with 0 results matches snapshot for no results', () => {
-  const { container } = render(<ResultsInfo resultsCount={0} />);
+  const { container } = render(
+    <ResultsInfo itemType="Venue" resultsCount={0} />
+  );
 
   expect(container).toMatchSnapshot();
 });
 
 it('renders no events found text', async () => {
-  render(<ResultsInfo resultsCount={0} />);
+  render(<ResultsInfo itemType="Venue" resultsCount={0} />);
```

Comment:
The test text does not match the result and the tested thing. Each of them could be easily tested here with `it.each<['General', 'Course', 'Venue']>`.